src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,12 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <div className="page-content">
+    <div className="movies">
```

Comment:
`div.page-content` does not concern component MovieList
you can move it to App component